packages/shopping/src/controllers/user-management.controller.ts

```diff
@@ -297,4 +306,70 @@ export class UserManagementController {
 
     return {token};
   }
+
+  @post('/users/reset-password/init', {
+    responses: {
+      '200': {
+        description: 'Confirmation that reset password email has been sent',
+      },
+    },
+  })
+  async resetPasswordInit(
+    @requestBody() resetPasswordInit: ResetPasswordInit,
+  ): Promise<string> {
+    if (!isemail.validate(resetPasswordInit.email)) {
+      throw new HttpErrors.UnprocessableEntity('Invalid email address');
+    }
+
+    const nodeMailer: NodeMailer = await this.userManagementService.requestPasswordReset(
```

Comment:
Maybe put a limit on how many times to allow the reset request in an hour/day? Else, one could flood someone's email by making 100s of reset requests.

Reply:
I will add another field on db table resetCount or something and limit it to 2. Does that sound like a sound solution?

Reply:
Yes two per hour sounds good. Also you will have to show an appropriate message when this limit is reached.

Reply:
@hacksparrow I think am missing something. You would prefer to limit the reset emails to 2 per hour in a day? or to 2 emails a day?

Reply:
Any would do, as long as we have a configurable setting.